src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/Builder.cs

```diff
@@ -130,9 +130,14 @@ string GetVisualStudio2017Directory ()
 			get {
 				string path;
 				if (IsUnix) {
-					path = Path.Combine ("/usr", "local", "share", "dotnet", "sdk", "2.0.0", "Sdks", "Microsoft.NET.Sdk");
-					if (File.Exists (Path.Combine (path, "Sdk", "Sdk.props")))
-						return path;
+					path = System.IO.Path.Combine ("/usr", "local", "share", "dotnet", "sdk");
+					var dirs = Directory.GetDirectories (path, "*.*.*").OrderByDescending (x => x);
```

Comment:
I have written code similar to this for xabuild, maybe could reuse it? https://github.com/xamarin/xamarin-android/blob/b78dd8bf4b1656ade4f9a46de767975400bc0101/tools/xabuild/XABuildPaths.cs#L227-L245

This looks like the code in this PR would order alphabetical, I think I was seeing that putting things in the wrong order when I wrote the above for xabuild. I think checking for `Sdk.props`, is probably a good idea, though.